source/lex.tex

```diff
@@ -1370,7 +1370,7 @@
 
 \pnum
 \enternote A source-file new-line in a raw string literal results in a new-line in the
-resulting execution \term{string-literal}. Assuming no
+resulting execution \grammarterm{string-literal}. Assuming no
```

Comment:
This should be "string literal", not "\grammarterm{...}" nor "\term{...}", because it's talking about the string literal object, not the element of the grammar.